app/src/main/java/de/westnordost/streetcomplete/quests/LastPickedValuesStore.kt

```diff
@@ -30,29 +30,37 @@ class LastPickedValuesStore<T : Any>(
 
 private const val MAX_ENTRIES = 100
 
-/* In the first `historyCount` items, return the `count` most-common non-null items, in order.
- * If the first item is not included (and is not null), it replaces the last of the common items.
- * If fewer than `count` unique items are found, continue counting items until that many are found,
- * or the end of the sequence is reached.
+/* Returns the `count` most-common non-null items in the first `historyCount`
+ *  items of the sequence, in their original order.
+ * If there are fewer than `count` unique items, continues counting items
+ *  until that many are found, or the end of the sequence is reached.
+ * If the first item is not null, it is always included, displacing the
+ *  least-common of the other items if necessary.
  */
 fun <T : Any> Sequence<T?>.mostCommonWithin(count: Int, historyCount: Int): Sequence<T> {
```

Comment:
Please rename `count` to `target` because

1. it's consistent with `countUniqueNonNull` then
2. it prevents confusion between `counts` and `count`

Reply:
I like `target` (and even `count`) way better than `n`.

Reply:
Done